packages/frontend/src/components/pagination/index.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { Dispatch, SetStateAction } from "react";
+import { PaginationBall } from "./pagination-ball";
+import { ReactComponent as ArrowIcon } from "../../assets/arrow.svg";
+import { cva } from "class-variance-authority";
+
+const createArrayBetweenRange = (start: number, end: number) =>
```

Comment:
I think that we can remove this function, since it's only used one time.

Reply:
In this case as it a bit of verbose code I do prefer to encapsulate in a function. 

Do you think we can left it like this or do you have a strong opinion on that?

Reply:
Let's leave it like this for now.